src/components/careers/Benefits/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import BenefitsCard from './Card'
+import ContentWrapper from '../../Layout/ContentWrapper'
+
+import BookIcon from '../../../assets/icon-book.svg'
+import BulbIcon from '../../../assets/icon-bulb.svg'
+import BuildingIcon from '../../../assets/icon-building.svg'
+import CalendarIcon from '../../../assets/icon-calendar.svg'
+import TeamIcon from '../../../assets/icon-team.svg'
+import LaptopIcon from '../../../assets/icon-laptop.svg'
+import TokensIcon from '../../../assets/icon-tokens.svg'
+import GlobeIcon from '../../../assets/icon-globe.svg'
+import BgGray from '../../../assets/bg-gray.svg'
+import { SectionTitle, Text } from '../styled'
+import { sizes } from '../../../styles/styleUtils'
+
+const Grid = styled.div`
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 24px;
+
+  @media screen and (max-width: ${sizes.md}px) {
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+    gap: 20px;
+  }
+`
+
+const SectionHeader = styled.div`
+  margin: 0 auto 80px;
+  max-width: 732px;
+  text-align: center;
+`
+
+const SBgGray = styled(BgGray)`
+  position: absolute;
+  top: 50px;
+  width: 2000px;
+  transform: translateX(-50%);
+  left: 50%;
+  min-width: 100%;
+
+  @media screen and (max-width: 980px) {
+    top: 300px;
+  }
+  fill: #e9e9e9;
+`
+
+const Section = styled.section`
+  background-color: ${(p) => p.theme.palette.white};
+  position: relative;
+  padding: 100px 0;
+  overflow: hidden;
+
+  @media screen and (max-width: ${sizes.md}px) {
+    padding: 64px 0;
+  }
+`
+
+const Benefits = () => {
+  return (
+    <Section>
+      <SBgGray />
+      <ContentWrapper>
+        <SectionHeader>
+          <SectionTitle>Our Benefits</SectionTitle>
+          <Text>
+            To make the lives of our employees better we developed some
+            innovative benefits. We work hard every day to make life of our
+            employees better.
```

Comment:
This is a bit repetetive. I'd rework it a bit to be something like:

```suggestion
We strive to make the lives of our employees better every day. As such, we have developed some innovative benefits for them.
```